test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -356,7 +347,6 @@ def setup_logging_stubs(should_fail = false, code = 0, message = 'Ok')
     end
     srv.stop
     t.join
-    @mock_host = nil
```

Comment:
We probably want to leave this here, so that `@mock_host` from the previous test doesn't leak into subsequent ones.

Reply:
Since we no longer check whether `@mock_host` is set before using it, we should not need to worry about setting it to `nil`, right? Each logging stub will use a new port anyway. We just need to make sure no more than 1 driver gets created in one logging stub context.

Reply:
If a driver gets created outside of the logging stubs context, we don't want it to reuse the port of the previous test, which is what would happen if we don't reset `@mock_host`, unless I'm missing something.
The second driver within the same stub context will fail during creation, because it'll try to bind to the same port, so I wouldn't worry about that.

Reply:
For a driver that gets created outside of the logging stubs context, we don't expect it to call the `api_client` method to use the port. If it does, it will fail, whether `mock_host` is `nil` or the port of the previous test. Maybe it's better to fail with `nil`. I just added the `@mock_host = nil` back.